### PR TITLE
[CDAP-9348] DataPrep add connections menu

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-9348

### CODE REVIEW

**Review by ajainarayanan — COMMENTED** [review]
`DataPrepConnections/index.js` file is missing i18n I guess.

File: cdap-ui/app/cdap/components/DataPrepConnections/index.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  return (
+    <Redirect to={`/ns/${namespace}/connections/browser`} />
+  );
+};
+
+const UploadPlaceholder = () => {
+  return (
+    <div>
+      <h3 className="text-xs-center">UPLOAD</h3>
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/DataPrepConnections/index.js:41]: You missed this in i18n

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 2]
```diff
@@ -369,6 +369,12 @@ features:
         uploadBtnLabel: Upload
     Upgrade:
       minimumVersionError: "Data Preparation requires wrangler-service artifact version 1.3.0 or above. Version {highestVersion} found. Please install the latest wrangler-service from Cask Market."
+
+  DataPrepConnections:
+    title: "Connections in \"{namespace}\""
+    upload: Upload
+    hdfs: HDFS
```
Comment (tonybach) [inline cdap-ui/app/cdap/text/text-en.yaml]: reorder?

**Review by tonybach — COMMENTED** [review]
Minor comments. Otherwise code LGTM 👍

File: cdap-ui/app/cdap/components/DataPrepConnections/index.js (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/DataPrepConnections/index.js:41]: That's just a placeholder, that component will be completely removed when we have an upload page

**Comment by tonybach** [comment]
Code LGTM 👍

**Comment by bdmogal** [comment]
I think the URL needs improvements. But that can be done separately. Other than that, 👍

**Pull request merged**